### ISSUE TITLE
API: map: empty map should be returned as [] instead of null

### ISSUE THREAD

**Issue by Frankkkkk** [haproxytech/dataplaneapi#234]
Hi,
This issue is related to #235 
The map API enables us to query the map files.

One example route is `GET /services/haproxy/runtime/maps_entries `, which, as per the doc, `Returns an array of all entries in a given runtime map file.`

However, when the map file is empty, the returned result in `null` instead of the null list (`[]`). This breaks tools that try to parse the json and thus can't infer the type from the null value.

Example:
```
$ curl -H "Content-Type: application/json" http://haproxy:5555/v2/services/haproxy/runtime/maps_entries?map=backend.map
$
```

PS: this issue could be extended to all routes that normally return a list of things but return null when empty

Cheers

**Comment by mjuraga** [comment]
Hi this is indeed a bug, we will fix it in the following versions. Thank you for the report.

**Comment by mjuraga** [comment]
This has been fixed in both v2.4.6 and v2.5.1, give it a try and let us know.
